SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.WindowsDesktop/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Newtonsoft.Json" version="6.0.1" targetFramework="net45" />
+</packages>
```

Comment:
I cannot comment on binaries itself so doing it in the file just before.

Why do we commit binaries to git?

* Tests/Microsoft.AppCenter.Crashes.Test.Windows/Fakes/System.fakes
* Tests/Microsoft.AppCenter.Crashes.Test.Windows/Fakes/mscorlib.fakes

Reply:
@guperrot 

GitHub doesn't recognize non standard `UTF-16` encoding.

`*.fakes` are text files encoded in `UTF-16 LE`, a special `UTF-16` with BOM according to Wikipedia (https://en.wikipedia.org/wiki/UTF-16).

> The standard also allows the byte order to be stated explicitly by specifying UTF-16BE or UTF-16LE as the encoding type. When the byte order is specified explicitly this way, a BOM is specifically not supposed to be prepended to the text, and a U+FEFF at the beginning should be handled as a ZWNBSP character. Most applications ignore a BOM in all cases despite this rule.

![image](https://user-images.githubusercontent.com/4079255/59728754-232f6d00-91f0-11e9-8ef0-c865d0603f77.png)

Reply:
Oh wow.
Ok 😅